src/components/Tooltip/Tooltip.vue

```diff
@@ -23,48 +29,60 @@
         props: {
             label: {
                 type: String,
-                required: true
             },
             placement: {
                 type: String,
-                default: 'bottom'
+                default: 'bottom',
             },
             disabled: {
                 type: Boolean,
-                default: false
+                default: false,
             },
             offset: {
                 type: String,
-                default: '0,5'
+                default: '0,5',
             },
             appendToBody: {
                 type: Boolean,
-                default: false
+                default: false,
             },
             boundariesElement: {
                 type: String,
-                default: 'viewport'
-            }
+                default: 'viewport',
+            },
+            delay: {
+                type: Number,
+                default: 0,
```

Comment:
I would say to go with the default `300ms`, as in Atlassian Design System, but I see the props of this component don't actually match the ADS 🤷‍♂️ so imo go with whatever you want.

Reply:
I don't think we should change the default behaviour